test/e2e/data/infrastructure-metal3/bases/cluster/crs.yaml

```diff
@@ -0,0 +1,24 @@
+---
+# ConfigMap object referenced by the ClusterResourceSet object and with
+# the CNI resource defined in the test config file
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "cni-${CLUSTER_NAME}-crs-0"
+data: ${CNI_RESOURCES}
```

Comment:
where are you defining this var `CNI_RESOURCES `?

Reply:
For future reference, the `e2e_cni.CNIPath` is the same as `CNI` in the e2e config. See https://github.com/kubernetes-sigs/cluster-api/blob/fa70f61a77828cea2d5d059ffd0dfcf6bcbc9f2d/test/e2e/common.go#L39